src/CsharpClient/Quix.Streams.Process.Samples/RandomDataProducer.cs

```diff
@@ -9,10 +9,10 @@ namespace Quix.Streams.Process.Samples
     /// <summary>
     /// Simply reader component that generates random tdata messages and writes them to the output.
     /// </summary>
-    public class RandomDataReader : StreamComponent
+    public class RandomDataProducer : StreamComponent
     {
 
-        public RandomDataReader()
+        public RandomDataProducer()
```

Comment:
Isn't the previous Reader the current Consumer?

Reply:
This generates data to an output, so I thought producer is more accurate. This could go either way, depending on context tbh